Print social links in speaker’s cards

rusty-key opened the pull request:
Hey!

During and after the conference, I wanted to look up speakers’ twitter/github links but there was no such info. This pull request fixes that problem. 

If you have some comments regarding styles I would gladly make changes in this PR. Also, I did use Font Awesome for social icons for the sake of speed. If it is not optimal, I'll include svg-icons to the build.

Result looks like this:
<img width="799" alt="screen shot 2018-05-15 at 15 37 34" src="https://user-images.githubusercontent.com/1197026/40057396-e061541c-5856-11e8-981e-41d326539869.png">

Commit:
print social links in speaker card

okonet commented:
First of all, thanks for this PR! This is highly appreciated and welcomed!

As for font awesome: we already have social icons package with types so it would be great if you could use it. 

As for design, can you please group those with the name. I’d try to put them above or believe the company. Can you try making them “in-line”?

Commit:
move links to the name

rusty-key commented:
@okonet, can you point in the direction of this social icons, please? I wasn't able to find them.

> As for design, can you please group those with the name. I’d try to put them above or believe the company. Can you try making them “in-line”?

I moved them, but they look too active now. I can remove color or decrease font-size a little bit to leverage "activeness". What do you think?

<img width="799" alt="screen shot 2018-05-15 at 21 03 26" src="https://user-images.githubusercontent.com/1197026/40075032-f5ce0994-5883-11e8-9c06-a667f9acc93d.png">

okonet commented:
@rusty-key here are the bindings: https://github.com/ReasonVienna/reason-conf/blob/master/src/gatsby.re#L63

What if we place them (with same color / style) after the company in one line? I think it's gonna be fine then.

okonet commented:
Actually, I think keeping red color for links can be good if we could make icons stand out less. But let's try changing the color for links as well?

Commit:
drop fontawesome, restyle social icons

rusty-key commented:
@okonet, moved icons to after the company name, removed text from links, and dropped fontawesome. What do you think?

<img width="831" alt="screen shot 2018-05-16 at 13 56 25" src="https://user-images.githubusercontent.com/1197026/40113197-4f835be6-5911-11e8-9b5e-0f72e952008e.png">

There was no icon for a website, so I used sharethis icon.

okonet commented:
I like it! Thanks again for your work!

okonet commented on src/layouts/layoutIndex.re:
```diff
@@ -71,7 +71,6 @@ let make = (~location, children) => {
       </div>
     | Normal =>
       <div className="page">
-        <Helmet title />
```
Not sure what's this is change is about... LGTM otherwise.

rusty-key replied:
Fixed: https://github.com/ReasonVienna/reason-conf/pull/73/commits/1f7c1d2d550fa1400a9dbdd55cbc9555d67794bc

Commit:
restore helmet

ryyppy commented:
Thanks @rusty-key 🎉

nikgraf commented:
So awesome @rusty-key, thanks 🙌
